### PR TITLE
bye bye, Ruby 2.1!

### PR DESCRIPTION
**Pull request by unak**
Please review my poor English.

### CODE REVIEW

File: en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+author: "usa"
+translator:
+date: 2017-04-01 00:00:00 +0000
+lang: en
+---
+
+We announce that all support of Ruby 2.1 series has ended.
+
+After the release of Ruby 2.1.10 at the end of March of the last year, the support of Ruby 2.1 series was in security maintenance phase.
+But, after one year has passed, this phase has ended.
```
Comment (stomar) [inline en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: Now, after ...

File: en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Therefore, at March 31st 2017, all support of Ruby 2.1 series has ended.
```
Comment (stomar) [inline en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: March 31, 2017, ...
of the Ruby ...

File: en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Bug and security fixes from more recent Ruby versions will no longer be backported to 2.1, and no further patch release of 2.1 will be released.
+We highly recommend that you upgrade to Ruby 2.4 or 2.3 as soon as possible.
+
+## About currently supported Ruby versions
+
+* Ruby 2.4 series
```
Comment (stomar) [inline en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: Newline missing below this line (else: "Ruby 2.4 series Currently in ..."

or definition list:

```
Ruby ...
: Currently...
  Foo...

```

File: en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  Currently in normal maintenance phase.
+  We will backport bug fixes and release with the fixes whenever necessary.
+  And, if a critical security issue will be found, we will release the urgent fix for it.
```
Comment (stomar) [inline en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: an urgent fix

File: en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* Ruby 2.3 series
+  Currently in normal maintenance phase.
+  We will backport bug fixes and release with the fixes whenever necessary.
+  And, if a critical security issue will be found, we will release the urgent fix for it.
+
+* Ruby 2.2 series
+  Currently in security maintenance phase.
+  We will never backport any bug fixes to 2.2 except security fixes.
+  If a critical security issue will be found, we will release the urgent fix for it.
+  We are planning to end the support of Ruby 2.2 series at the end of March of 2018.
```
Comment (stomar) [inline en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: of the Ruby...

**Comment by unak** [comment]
Thank you very much, @stomar !
I've applied your comments.

**Pull request merged**